src/UI/Component/Panel/Sub.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel;
+
+/**
+ * This describes a Sub Panel.
+ */
+interface Sub extends Panel {
```

Comment:
Yes I could. What I am asking is: Do we need a Component named "Panel->WithContent" if we introduce this interface. I hope not.